x-pack/plugins/monitoring/public/application/pages/beats/instance.tsx

```diff
@@ -0,0 +1,101 @@
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useContext, useState, useCallback, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { i18n } from '@kbn/i18n';
+import { find } from 'lodash';
+import { ComponentProps } from '../../route_init';
+import { GlobalStateContext } from '../../global_state_context';
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
+import { useCharts } from '../../hooks/use_charts';
+// @ts-ignore
+import { Beat } from '../../../components/beats/beat';
+import { BreadcrumbContainer } from '../../hooks/use_breadcrumbs';
+import { PageTemplate, TabMenuItem } from '../page_template';
+
+export const BeatsInstancePage: React.FC<ComponentProps> = ({ clusters }) => {
+  const { instance }: { instance: string } = useParams();
+
+  const globalState = useContext(GlobalStateContext);
+  const { services } = useKibana<{ data: any }>();
+  const { generate: generateBreadcrumbs } = useContext(BreadcrumbContainer.Context);
+  const { zoomInfo, onBrush } = useCharts();
+  const clusterUuid = globalState.cluster_uuid;
+  const ccs = globalState.ccs;
+  const cluster = find(clusters, {
+    cluster_uuid: clusterUuid,
+  }) as any;
+  const [data, setData] = useState({} as any);
+  const [beatName, setBeatName] = useState('');
+
+  const title = i18n.translate('xpack.monitoring.beats.instance.routeTitle', {
+    defaultMessage: 'Beats - {instanceName} - Overview',
+    values: {
+      instanceName: beatName,
+    },
+  });
+
+  const pageTitle = i18n.translate('xpack.monitoring.beats.instance.pageTitle', {
+    defaultMessage: 'Beat instance: {beatName}',
+    values: {
+      beatName,
+    },
+  });
+
+  useEffect(() => {
+    if (cluster) {
+      generateBreadcrumbs(cluster.cluster_name, {
+        inBeats: true,
+        instance: beatName,
+      });
+    }
+  }, [cluster, beatName, generateBreadcrumbs]);
+
+  const getPageData = useCallback(async () => {
+    const bounds = services.data?.query.timefilter.timefilter.getBounds();
+    const url = `../api/monitoring/v1/clusters/${clusterUuid}/beats/beat/${instance}`;
+    const response = await services.http?.fetch(url, {
+      method: 'POST',
+      body: JSON.stringify({
+        ccs,
+        timeRange: {
+          min: bounds.min.toISOString(),
+          max: bounds.max.toISOString(),
+        },
+      }),
+    });
+
+    setData(response);
+    setBeatName(response.summary.name);
+  }, [ccs, clusterUuid, instance, services.data?.query.timefilter.timefilter, services.http]);
+
+  const tabs: TabMenuItem[] = [
+    {
+      id: 'overview',
+      label: i18n.translate('xpack.monitoring.beatsNavigation.instance.overviewLinkText', {
+        defaultMessage: 'Overview',
+      }),
+      route: `/beats/beat/${instance}`,
+    },
+  ];
+
+  return (
+    <PageTemplate
```

Comment:
I think I might update this to do conditional tabs on BeatsTemplate. Using PageTemplate directly here works fine since beats only has one tab, but other parts of stack monitoring will have multiple tabs so will need a different pattern.